src/workerhelper.js

```diff
@@ -1,9 +1,9 @@
-var window = window || {},
+var win = window || {},
```

Comment:
we probably want to do `typeof window !== 'undefined' ? window : {}` so that it doesn't break in node, just in case.